packages/@glimmer/syntax/lib/builders.ts

```diff
@@ -572,6 +569,51 @@ export default {
   },
 };
 
+const locationBuilders = {
+  loc: buildLoc,
+  pos: buildPosition,
+};
+
+const modernBuilders = {
+  MustacheStatement: legacyBuilders.mustache,
```

Comment:
These should be camel case with initial lower case character. That is what recast and jscodeshift use (by way of [ast-types](https://github.com/benjamn/ast-types)).

For example, this one would be `b.mustacheStatement(...args...)`.

Reply:
I'm confused with babel builders

https://astexplorer.net/#/gist/6fa4e6380a5d53d6385efb7716012ba0/390e735ebaea048f40b7ecbc058c544cc58302df

Looks like they support both cases

Reply:
@rwjblue c builder aliases added (amel case with initial lower case character), should I remove capitalized?

Reply:
Hmm. Who uses initial capital case vs who supports initial lower? I’d really prefer not to have three different builder functions for the same thing...

Reply:
@rwjblue how about `Program/program` builder, should we have it? (it's return "Template" type)

Reply:
I'd prefer to only create _new_ aliases for the "go forward" AST types.

Reply:
@rwjblue the usual convention is that UpperCase is for class names and lowerCase for functions. so a factory function for `FooBar` would be called `fooBar()` and use `new FooBar()` internally.

tl;dr I agree that we should only use camelCase function names